CHANGELOG.md

```diff
@@ -17,6 +17,8 @@
 
 #### Changes
 
+* Extend test matrix with: (#235)
```

Comment:
```suggestion
* Migrate CI to GitHub Actions and extend test matrix with: (#235)
```

Reply:
I used this, but dropped "and extend test matrix with".  I've removed the clutter of extending the test matrix from this PR.